Templates/CSharp/Requests/EntityClient.cs.tt

```diff
@@ -39,6 +40,16 @@ namespace <#=entityContainer.Namespace.GetNamespaceName()#>
             : base(baseUrl, authenticationProvider, httpProvider)
         {
         }
+
+        /// <summary>
+        /// Instantiates a new <#=clientName#>.
+        /// </summary>
+        /// <param name="httpClient">Http client to use in making requests</param>
```

Comment:
Let's encourage customers to use the HttpClient from the GraphClientFactory. I suggest an update to the param doc to be:

`The HttpClient to use for making requests to Microsoft Graph. Use the GraphClientFactory to get a pre-configured HttpClient that is optimized for use with the Microsoft Graph service API.`

These comments are captured for IntelliSense.

Reply:
Great suggestion. Should be resolved now.